py3status/modules/xsel.py

```diff
@@ -66,7 +66,7 @@ def xsel(self):
         selection = " ".join(selection.split())
         if len(selection) >= self.max_size:
             if self.symmetric is True:
-                split = int(self.max_size / 2) - 1
+                split = (self.max_size // 2) - 1
```

Comment:
I guess we can loose the parenthesis too?

Reply:
Oh yes, thanks, fixed, repushed.